plugins/faustwp/includes/detect-conflicts/callbacks.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Callbacks for display and dismissal of plugin conflicts warning.
+ *
+ * @package FaustWP
+ */
+
+namespace WPE\FaustWP\Detect_Conflicts;
+
+/** Unique ID for the conflict warning. */
+const NOTICE_ID = 'faustwp_plugin_conflicts';
+
+/**
+ * Renders the plugin conflicts admin notice.
+ *
+ * @return void
+ */
+function show_warning() {
+	if ( ! needs_warning() ) {
+		return;
+	}
+
+	$plugins  = get_plugin_conflicts();
+	$warnings = array();
+
+	foreach ( $plugins as $plugin ) {
+		$plugin_headers = get_plugin_data( WP_PLUGIN_DIR . '/' . $plugin );
+		$warnings[]     = $plugin_headers['Name'];
+	}
+
+	printf(
+		'<div id="%s" class="notice notice-warning notice-alt is-dismissible">
+			<h2 class="notice-title">%s</h2>
+			<p>%s</p>
+			<p>%s</p>
+		</div>',
+		esc_attr( NOTICE_ID ),
+		esc_html__( 'Incompatible plugins detected', 'faustwp' ),
+		esc_html__( 'The following active plugins are known to conflict with FaustWP and may cause unexpected behavior:', 'faustwp' ),
```

Comment:
We may want to consider adding FaustWP to the notice title in addition to paragraph.

"FaustWP Detected Incompatible Plugins"

<img width="1137" alt="Screen Shot 2022-04-25 at 11 29 53 AM" src="https://user-images.githubusercontent.com/6676674/165123471-2f7c4214-b5d8-47b1-bcc8-6a3ed7aebaee.png">